package com.bytedance.sdk.component.xha.p169ik;

import android.text.TextUtils;
import java.util.Map;

/* JADX INFO: renamed from: com.bytedance.sdk.component.xha.ik.ri */
/* JADX INFO: loaded from: classes3.dex */
public class C2738ri {
    /* JADX WARN: Code duplicated, block: B:10:0x0023 A[PHI: r1
  0x0023: PHI (r1v1 java.lang.String) = (r1v0 java.lang.String), (r1v3 java.lang.String) binds: [B:6:0x0010, B:8:0x001e] A[DONT_GENERATE, DONT_INLINE]] */
    /* JADX INFO: renamed from: ri */
    public static boolean m10353ri(Map<String, String> map) {
        String str;
        if (map == null) {
            return false;
        }
        String str2 = "content-type";
        if (TextUtils.isEmpty(map.get("content-type"))) {
            str2 = "Content-Type";
            if (TextUtils.isEmpty(map.get("Content-Type"))) {
                str = "";
            } else {
                str = map.get(str2);
            }
        } else {
            str = map.get(str2);
        }
        return str.contains("application/octet-stream");
    }
}
